### ISSUE TITLE
Redirect on Successful License Submission

### ISSUE THREAD

**Issue by BassCoder2808** [spdx/spdx-online-tools#449]
Whenever we submit a new license, we get redirected to the same page after successful submission, instead, it should take the user to the issues page as discussed with @jlovejoy.

**Comment by jlovejoy** [comment]
okay, I just tested this b/c I couldn't remember (and sometimes I don't notice these things because I just know where to go!) and yeah, it should not just stay on the submission page. I think going to the newly created GitHub issue would be best.

**Comment by jlovejoy** [comment]
as discussed, have newly created Github issue open in new tab/window

**Comment by BassCoder2808** [comment]
Yes, I had made changes to the PR as well #450, @rtgdk we can merge that if nothing else is needed in that

**Comment by jlovejoy** [comment]
ok, we'll close issue once PR is merged

**Comment by jlovejoy** [comment]
so, this was added, but the new window was blocked by my browser's pop-up blocker... maybe we should consider how common that might be?

**Comment by BassCoder2808** [comment]
Hi @jlovejoy yes for the first time the pop-up window will be blocked for everyone by the browser itself, but once you allow it, it will not ask again. Actually, this is a security feature in every browser by default, so for the first time the user will need to allow pop-ups from `tools.spdx.com` to open

**Comment by goneall** [comment]
@jlovejoy @BassCoder2808 - I'm going to close this as the original issue was resolved with PR #450 

If the pop-up blocker is an issue, please open a new issue.
